perseo-utils/src/main/java/com/telefonica/iot/perseo/utils/DateTimeUtils.java

```diff
@@ -3,17 +3,23 @@
  *
  * This file is part of perseo-core project.
  *
- * perseo-core is free software: you can redistribute it and/or modify it under the terms of the GNU
- * General Public License version 2 as published by the Free Software Foundation.
+ * perseo-core is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU
+ * General Public License version 2 as published by the Free Software
+ * Foundation.
  *
- * perseo-core is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
- * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * perseo-core is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License
  * for more details.
  *
- * You should have received a copy of the GNU General Public License along with perseo-core. If not, see
+ * You should have received a copy of the GNU General Public License
+ * along with perseo-core. If not, see
  * http://www.gnu.org/licenses/.
  *
- * For those usages not covered by the GNU General Public License please contact with
+ * For those usages not covered by the GNU General Public License
+ * please contact with
```

Comment:
Please don't change existing licence headers.

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823